src/states/game/game.gd

```diff
@@ -19,6 +20,7 @@ var blue_max_health: int = 200
 
 
 func _ready() -> void:
+	options_menu.get_node("CenterContainer/VBoxContainer/BackButton").pressed.connect(hide_options)
```

Comment:
I'd prefer that `options_menu` exposes a signal rather than digging into it and finding the button, but making `BackButton` a scene unique name and using `%BackButton` would be acceptable